MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -969,15 +979,18 @@ private void BuildStreamVideoItem(
                 // Reverse codec profiles for backward compatibility - first codec profile has higher priority
                 .Reverse();
 
-            foreach (var i in appliedVideoConditions)
+            if (videoCodec is not null)
```

Comment:
I think this check will become unnecessary when https://github.com/jellyfin/jellyfin/pull/12521 is backported.